modules/ticket_selector/assets/ticket_selector.js

```diff
@@ -56,7 +56,14 @@ jQuery(document).ready(function ($) {
                         $ticket_selector_row.removeClass('ee-hidden-ticket-tr');
                         active_rows++;
                     } else {
-                        $ticket_selector_row.addClass('ee-hidden-ticket-tr').find('.ticket-selector-tbl-qty-slct').val(0);
+						$ticket_selector_row.addClass( 'ee-hidden-ticket-tr' );
+						$qty_input = $ticket_selector_row.find(
```

Comment:
It doesn't look like `$qty_input` has been declared anywhere.  In js, it's best practice to do so.